build error in `errors`

https://pluralsight.slack.com/archives/C70HPSJPP/p1603825765135600

```
ERROR in ./node_modules/@pluralsight/ps-design-system-errors/dist/esm/react/icon-loader.js
Module not found: Error: Can't resolve 'import-all.macro' in '/Users/nate-piche/work/workflow-skills-development/ps-time/ui/node_modules/@pluralsight/ps-design-system-errors/dist/esm/react'
```

--- a/packages/errors/package.json
+++ b/packages/errors/package.json
@@ -10,9 +10,9 @@
   "module": "dist/esm/index.js",
   "scripts": {
     "build": "run-s build:svg build:esm build:cjs build:css",
-    "build:cjs": "tsc --project tsconfig.build.json --module commonjs --target es5 --outDir dist/cjs",
+    "build:cjs": "babel --extensions \".js,.ts,.tsx\" --root-mode upward src --out-dir dist/cjs",
     "build:css": "build-css --useGlamor -i dist/cjs/css/index.js",
-    "build:esm": "tsc --project tsconfig.build.json --module es2015 --target es5 --outDir dist/esm",
+    "build:esm": "cross-env ESM=true babel --extensions \".js,.ts,.tsx\" --root-mode upward src --out-dir dist/esm",
     "build:svg": "svgr --filename-case kebab --ext \"dist.js\" -d src/react/icons src/svg",
     "build:watch": "yarn build:js -- --watch",
     "storybook": "start-storybook -p 6006",

